payments-model/src/main/java/com/stripe/android/model/CardBrand.kt

```diff
@@ -45,6 +45,30 @@ enum class CardBrand(
      */
     private val variantMaxLength: Map<Pattern, Int> = emptyMap(),
 ) {
+    Visa(
```

Comment:
Is this changed so that the order appears correctly in the UI?  If so it might be better to have a getOrderedBrandList function.  I can see adding a brand, and then not realizing I was impacting functionality, or changing the order around in the enum.  The test would check it, but it might be better to be explicit.  If there was a question you could look at the caller of getOrderedBrandList and see how it was being used.

Reply:
The one thing I don't like is that we will need to maintain this list if there is a new `CardBrand` added, I think it can be easily missed.

Reply:
In this situation I feel like it's better to have an explicit list. Your test could also assert that the orderedBrandList function has the same length as `CardBrand.values()`

Reply:
I don't really see the difference between having a test that tests the order, and this. It will get caught either way.

Reply:
I added a new field to `CardBrand` called `renderingOrder` let me know what you think of this approach

Reply:
sounds great, thanks James!